src/context/meta-tags-context.php

```diff
@@ -352,24 +351,6 @@ public function generate_site_represents_reference() {
 		return false;
 	}
 
-	/**
-	 * Generates whether or not breadcrumbs are enabled.
-	 *
-	 * @return bool Whether or not breadcrumbs are enabled.
-	 */
-	public function generate_breadcrumbs_enabled() {
```

Comment:
@omarreiss You delete this public function without deprecating it. Is that intentional?

Reply:
Done in 39b6582f5081923f52c3c756924e20d3d9d2b7a7